rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -108,6 +109,16 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/Version'
+    Last-Modified:
+      name: Last-Modified
+      in: header
+      description: >
+        The last modified time of the product to be ingested.
```

Comment:
✏️ Can we say "of the intel" instead of the product since at some point, the product will be optional?